content/800-data-platform/300-cloud-projects/200-platform/200-projects.mdx

```diff
@@ -188,69 +188,52 @@ See [Usage](/data-platform/cloud-projects/platform/organizations#usage) for deta
-
-</Admonition>
-
-Alternatively, if you have been given access to Pulse, you can click **Configure Pulse** under the **Get Started** section of your project's **Dashboard** page.
+> This option will look slightly different if you have not yet been given Early Access to Pulse. In that scenario the link will prompt you to join the waitlist.
 
 On this page, there are two important sections with instructions to follow to set up Pulse:
 
-#### Prepare your database
+### Prepare your database
 
 First is the **Prepare your database** section. Here, a set of instructions are defined that guide you on preparing your database for supporting real-time functionality.
 
-These configurations are not Prisma-specific, but rather are database configurations Pulse relies on. For more detailed instructions, see the [Pulse documentation](/data-platform/pulse/what-is-pulse).
+These configurations are not Prisma-specific, but rather are database configurations Pulse relies on. For more detailed instructions, see the [Pulse documentation](https://docs-git-mhess-pulse-docs-updates-prisma.vercel.app/data-platform/pulse/what-is-pulse).
 
-<Admonition type="info">
-
-Pulse currently works with PostgreSQL databases and MySQL support is coming soon.
-
-</Admonition>
+> Pulse currently works with PostgreSQL databases and MySQL support is coming soon.
 
-#### Configure Pulse
+#### Enable Pulse
 
 This section will prompt you for two pieces of information needed to set up Pulse:
 
 1. Your database connection string
 2. The region where you would like to host Pulse
 
-<Admonition type="info">
-
-It is recommended to choose a region closest to your database as this will minimize latency between Pulse and the database itself.
-
-</Admonition>
+> It is recommended to choose a region nearest your database as this will minimize latency between Pulse and the database itself.
+>
+> Once those fields are filled in, click **Enable Pulse** at the bottom of the form. This will configure Pulse to be used by your project.
 
-Once those fields are filled in, click **Configure Pulse** at the bottom of the form. This will configure Pulse to be used by your project.
-
-After this completes, the view will display [instructions](/data-platform/pulse/what-is-pulse) on how to begin using Pulse in your application.
+After this completes, the view will display [instructions](/data-platform/pulse/what-is-pulse) on how to add Pulse to your application and start your first subscription.
 
 Specifically, you will be prompted to:
 
-1. Get a [Project API Key](/data-platform/cloud-projects/platform/projects#project-api-keys) if you do not already have one to use in the Pulse extension
-2. Set up Pulse in your application
+1. Add a [Project API Key](/data-platform/cloud-projects/platform/projects#project-api-keys) if you do not already have one to use in the Pulse extension
+2. Add Pulse to your application
 
-### Disable Pulse
+#### Disable Pulse
```

Comment:
```suggestion
### Disable Pulse
```

I think this should be kept as a `h3` since it's not part of the "Prepare your database" section.

Reply:
The issue was the right side nave when these were h3's
![image](https://github.com/prisma/docs/assets/36578389/cd3f2068-5c05-48a4-9b10-255ef3f66dfa)

Let me know what you think. Ill commit this change if it doesn't matter.

Reply:
The TOC shows the hierarchy of the document, so imo this should look like:

- Pulse
  - Setup Pulse
    - Prepare your database (not visible in the TOC)
    - Enable Pulse (not visible in the TOC)
  - Disable Pulse
  - View Pulse usage